content/800-guides/240-partner-api.mdx

```diff
@@ -0,0 +1,163 @@
+---
+title: 'Partner Database Provisioning & User Claim Flow'
+metaTitle: 'Partner Database Provisioning & User Claim Flow'
+description: 'Learn how to use the Prisma Postgres Partner API to provision and claim databases'
+sidebar_label: 'Partner API'
+image: '/img/guides/prisma-partner-api-cover.png'
+community_section: true
+---
+
+## Introduction
+
+This guide walks you through how to use the Prisma Postgres Partner API, which powers the [`npx create-db`](https://create-db.prisma.io/) command.
+
+You'll learn how to provision a Prisma Postgres database on your workspace as a partner, and how to transfer it to another user's workspace so they can "claim" the database. We'll cover how the process is secured using OAuth2, and by the end, you'll understand the full flow and how to integrate it into your own product experience.
+
+This guide references the actual implementation in the `npx create-db` CLI and Cloudflare Workers as real world examples. The repo for the `npx create-db` is [here](https://github.com/prisma/create-db), which can be used as a reference for how to use the Partner API in your own projects.
+
+:::note How does this fit into your app?
+The two workers in this guide are just reference examples. You would typically build this logic into your own backend or serverless functions.
+
+Similarly, the `npx create-db` CLI is a simple demo. In your product, you can trigger the same API calls from your own UI or onboarding flows to create a seamless experience for your users.
+:::
+
+## Core Concepts
+
+Before diving into implementation, let's clarify the main concepts involved in the Partner API integration:
+
+- **Partner API**: A set of endpoints that allow approved partners to provision and manage Prisma Postgres databases on behalf of their users.
+- **Projects & Databases**: In Prisma Postgres, a "project" represents a database instance. When you provision a project, you are creating a new database for your user.
```

Comment:
That's not right. A database is not the same as a project, a project can contain multiple databases.